src/metabase/sync/analyze/fingerprint/fingerprinters.clj

```diff
@@ -185,6 +186,14 @@
   String   (->temporal [this] (u.date/parse this))
   Long     (->temporal [this] (t/instant this))
   Integer  (->temporal [this] (t/instant this))
+  ;; this is challenging, because ChronoLocalDate requires a ZoneOffset
+  ;; to work with. Ideally, we would use the database's ZoneOffset, but
+  ;; we don't have access to that here. Use the JVM's systemDefault to
+  ;; convert this.
+  ChronoLocalDateTime (->temporal [this] (.toInstant this
```

Comment:
if someone has an idea of the best way to get the database's timezone here, I'm all ears

Reply:
Have a look at `query_processor.timezone` if you can use something from there

Reply:
I started with that, but it complains about the query processor not being initialized. I'm not sure how to do that.

Reply:
I think that's only with tests, when this runs for real it's a part of the QP and the store should be initialized already (just reverse engineer which bits of the QP store you need for it to work).